src/debugger/Debugger.js

```diff
@@ -127,6 +131,37 @@ export class DebugServer {
     }
   }
 
+  checkStepIn(ast: BabelNode) {
+    if (this._steppingInfo !== undefined) {
+      if (this.isValidSteppingLocation(ast)) {
+        if (ast.loc && ast.loc.source) {
+          this._channel.sendStepInResponse(ast.loc.source, ast.loc.start.line, ast.loc.start.column);
+          this._steppingInfo = undefined;
+          this.waitForRun(ast);
+        }
+      }
+    }
+  }
+
+  isValidSteppingLocation(ast: BabelNode): boolean {
```

Comment:
The name does not make sense. The meaning of this method should be "shouldCompleteStep()" or something similar.